github_actions/spec/fixtures/vcr_cassettes/Dependabot_GithubActions_UpdateChecker/_can_update_/given_a_dependency_with_a_tag_reference/with_a_dependency_that_has_a_latest_requirement_and_a_valid_version/1_2_2_6_1.yml

```diff
@@ -0,0 +1,91 @@
+        refs/pull/53/merge\n003f64c9a1fc5f8bb3bd806fff9fbf32ac9f6e25310e refs/pull/54/head\n0040df2fc049a27f1d4e627c6ac1d8f9cb16d31b8780
+        refs/pull/54/merge\n003ff5ba677a6062d133579ec5b78ca6e6592e45bc52 refs/pull/55/head\n0040439cb5df639f2a1a56194ffb55f5adfe8d56ebd5
+        refs/pull/55/merge\n003fe28d56d234524918dec50dff7281be367f54d293 refs/pull/57/head\n003f131b819361b31e0d001f3d08c10208691181c6b0
+        refs/pull/60/head\n00409849cc8cb28428620a5dcd23e52538c302e52057 refs/pull/60/merge\n003fdcf11047e5018d606270008ed618edf5514d30bd
+        refs/pull/61/head\n00401e946e8f12b0d22717c5d825f06b50cbb72363b7 refs/pull/61/merge\n003f6addae869b866c42918eb511cdbecb166c56b572
+        refs/pull/62/head\n003fac94e9da978f84c77ee126a186cbc658def378db refs/pull/66/head\n00402e1c38f45c9eae25c4853c35d245428705be9c9f
+        refs/pull/66/merge\n003f26fb39e737862250a3c4710b3698adc44dc90952 refs/pull/69/head\n00404c76da40e3ce4ef5bcef6f41f6e5b55bb73ab7b8
+        refs/pull/69/merge\n003f7bc451acb8d5fc227f6718dfeb1935dcceb0ea15 refs/pull/70/head\n003fec9ee66a820ce06f346c7ed3301ae58b1046e74f
+        refs/pull/73/head\n003f80985fc475a01e84fe82cab48437c59daa46093d refs/pull/75/head\n003f91b6c4e35327a5cb9beafc16fb70101ac4340c57
+        refs/pull/76/head\n003e2470c6ab61a23591b63fdbd128e0254070cd4b10 refs/pull/8/head\n003f44853f236dc6b4a530d8f4a86fc3ccd439566d69
+        refs/pull/81/head\n003fc068e8508ed8873161eb704b5e23b6b6a64ce37e refs/pull/82/head\n003fe5c76ffb5afe9e27434f93429a560e27d2829d44
+        refs/pull/84/head\n0040924eac954ce38e1256ddbfae7411f393c5966060 refs/pull/84/merge\n003f1a59e25dfa5cc4ce4e20914d0ab58102a9049aec
+        refs/pull/85/head\n0040c3f69b19844027b3ca299f9a835ea00e0d4561bd refs/pull/85/merge\n003f653bc6fb0229fa545fde680ae5e685ab99822130
+        refs/pull/86/head\n003f2c8f71882f7df483f8c60077e839dcd77c4bb110 refs/pull/87/head\n003f22098c348e61d788892de2c5a2bd3dc3046cfd35
+        refs/pull/88/head\n003f52f35f70f8db19ff5850f8a454bf87acb2af92d6 refs/pull/89/head\n0040eb0228e41914eb033762292e266e6ca5ffe5ab75
+        refs/pull/89/merge\n003f70321bad373c97665e2d7e463a4373ea17586c22 refs/pull/90/head\n003faa8d5ff2a35e819db0332788c64c03f00abf73ca
+        refs/pull/93/head\n0040ed3a08a68b8d83e5f69b4fe612dc1d60129de4e0 refs/pull/93/merge\n003f5a98c8fe6feae865a4da24d3042e5813c6c0688a
+        refs/pull/94/head\n00406a10cbc405cf4a95a328e41b382379c015605a6f refs/pull/94/merge\n003ff60d8eac1114608aba7746762ea1575ab4313dbd
+        refs/pull/95/head\n004048a8b21925e962d8a7738d323d915b0cecfe1f2b refs/pull/95/merge\n003f8868c043d3b524cc5bda7bfb5f1c205aee14b8b3
+        refs/pull/96/head\n003f7132fc2fa62693d51d29209dfac800bb3a92c358 refs/pull/97/head\n0040b581cd874bd08b678bc8cac29e2538bdb766e22b
+        refs/pull/97/merge\n003fc1929ead8889aba340647d4ae8a388c61b198424 refs/pull/98/head\n003e0cb9c9b65d5d1901c1f53e5e66eaf4afd303e70e
```

Comment:
To make it easier to spot the defect would it be helpful to delete the refs listed above. I think the only relevant refs are the ones listed below because the test indicates a ref of `@v1`. 

I'm also a bit confused by the data below, it looks like there is a `refs/tags/v1\n` with a sha1 (003e8d93430eddafb926c668181c71f579556f68668c) that matches the `refs/tags/latest` sha1.

Reply:
Hm I'm not sure we want to start editing vcr records as they'll all be regenerated from actual requests by running `VCR=true rspec spec`.

I think `v1` == `latest` is a common pattern, means you can just do `uses: actions/create-release@v1` to get the latest v1 release, so `v1 == v1.1.4 == latest` are all pointing to the same sha.